Add anthos known issue for disk deletion and 2.5.1.3 RN (#834)
portworx/pxdocs: content/reference/release-notes/portworx.md

series: release-notes
---

## 2.5.1.3

June 05, 2020

## Fixes

The following issues have been fixed:

|**Issue Number**|**Issue Description**|
|----|----|

> |PWX-13086| For vSphere 6.7.3 and above, create PX disks (vmdks) such that they don't get deleted on VM deletion. |

Reviewer: Is this still accurate?
```suggestion
|PWX-13086| For vSphere 6.7.3 and above, VMDKs created by Portworx no longer get deleted on VM deletion. |
```

Reviewer 2: For vSphere 6.7.3 and above, VMDKs/disks created by Portworx no longer get deleted on VM deletion.
